cpp/CMakeLists.txt

```diff
@@ -339,16 +355,17 @@ message(STATUS "CUDF_TEST_LIST set to: ${CUDF_TEST_LIST}")
 # - add google benchmark --------------------------------------------------------------------------
 
 if(BUILD_BENCHMARKS)
+    find_package(benchmark)
 
-  include(ConfigureGoogleBenchmark)
-
-  if(GBENCH_FOUND)
-    message(STATUS "Google C++ Benchmarking Framework (Google Benchmark) found in ${GBENCH_ROOT}")
-    include_directories(${GBENCH_INCLUDE_DIR})
-    add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
-  else()
-    message(AUTHOR_WARNING "Google C++ Benchmarking Framework (Google Benchmark) not found: automated tests are disabled.")
-  endif(GBENCH_FOUND)
+    if(benchmark_FOUND)
+        message(STATUS "Google C++ Benchmarking Framework (Google Benchmark) found in ${benchmark_ROOT}")
+        add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
+    else()
+        message(AUTHOR_WARNING "Google C++ Benchmarking Framework (Google Benchmark) not found, it will be built from source.")
+        include(ConfigureGoogleBenchmark)
+        add_subdirectory(${CMAKE_SOURCE_DIR}/benchmarks)
```

Comment:
I think we need to run this regardless of whether we find or build gbench

Reply:
My response for this is the same as above relating to GTEST.